### PR TITLE
Admin chargify subscription link

### PR DESCRIPTION
**Pull request by HSalmon**
[Trello card](https://trello.com/c/2BjWcOra/1872-2-link-to-chargify-subscription-page-from-group-loomio-admin-page)

chargify_subscription_url

### CODE REVIEW

File: app/admin/subscriptions.rb [hunk 1]
```diff
@@ -14,6 +14,24 @@
...
+    attributes_table do
+      row :id
+      row :kind
+      row :expires_at
+      row :trial_ended_at
+      row :activated_at
+      row :chargify_subscription_id do |subscription|
+        if subscription.chargify_subscription_id
+          link_to subscription.chargify_subscription_id,
+                  ChargifyService.new(subscription.chargify_subscription_id).subscription_url, target: '_blank'
```
Comment (gdpelican) [inline app/admin/subscriptions.rb]: I don't think I quite see the merit of including ChargifyService in this. Options:

- Create a helper method, `chargify_subscription_url(subscription)`
- Put a `chargify_url` method on subscription.rb, `subscription.chargify_url`
- Create a SubscriptionDecorator which wraps a subscription `subscription.decorate.chargify_url` (most correct, but we don't do this anywhere else yet.)

I'd reckon #1 would be best for now.

File: app/helpers/subscription_helper.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module SubscriptionHelper
+
+  def chargify_subscription_url(subscription)
+    ChargifyService.new(subscription.chargify_subscription_id).subscription_url
```
Comment (gdpelican) [inline app/helpers/subscription_helper.rb]: Heh, I think you misunderstood me; what I was thinking was

```
def chargify_subscription_url(subscription)
  return unless Rails.application.secrets.chargify_app_name
  "http://#{Rails.application.secrets.chargify_app_name}.chargify.com/subscriptions/#{subscription.chargify_id}"
end
```

That way we don't have to add those methods into the service.

Comment (gdpelican) [inline app/helpers/subscription_helper.rb]: You could even pull the whole link_to out into a helper

```
def chargify_subscription_url(subscription)
  return unless Rails.application.secrets.chargify_app_name
  link_to subscription.chargify_id, "http://#{Rails.application.secrets.chargify_app_name}.chargify.com/subscriptions/#{subscription.chargify_id}"
end
```

**Comment by HSalmon** [comment]
@gdpelican, I think I've implemented your feedback :)

**Comment by gdpelican** [comment]
Looks good to me, thanks! 🍏  I just put in a check to make sure we don't show the link if there's no subscription id

**Pull request merged**